### ISSUE TITLE
VSCode-Insiders "Open With Insiders" doesn't show up in Windows new Context Menu

### ISSUE THREAD

**Issue by DGamer007** [microsoft/vscode#166365]
I have installed VSCode-Insiders on my Windows PC. I know that the Insider build of VSCode has "Open with insiders" option in Windows new Context menu. Now, the problem is that it somehow doesn't show up in Context menu. Last I had installed VSCode-Insiders 3 days ago and at that time I was able to see that option in Windows Context Menu. But today I am unable to see it. I have tried restarting my PC. Didn't work.

**Comment by deepak1556** [comment]
Does this issue still repro with latest insiders ?

**Comment by leonardder** [comment]
I just installed the latest insiders build (2022-12-05T10:17:57.998Z) and even after a fresh install, I"m not getting the context menu option. I must say that I have a Windows 11 insiders preview running.

**Comment by DGamer007** [comment]
> I just installed the latest insiders build (2022-12-05T10:17:57.998Z) and even after a fresh install, I"m not getting the context menu option for folders. For files, it works fine. I must say that I have a Windows 11 insiders preview running.

Did you select the option for Context Menu in Installer? Coz after a fresh install I don't have any problem.

**Comment by leonardder** [comment]
Yes, I did. I assume it should somehow be possible to install a 1.72 insiders preview to compare the outcome.

**Comment by deepak1556** [comment]
Might be related to the insiders version of Windows, I am not able to repro the bug in Windows 11 Build 22621.819. 

> I'm only getting the Add "Open with Code - Insiders" action to Windows Explorer file context menu check box.

For the new context menu, this checkbox covers both files and folders.

**Comment by leonardder** [comment]
All right, I can confirm that it works now. I always open the windows 11 context menu using the keyboard (applications key). Somehow on my system, the Windows 11 specific context menu items do not appear. I just tried this on a fresh Windows 11 and there it works correctly. Visual Studio code does show up when I use the mouse to open the context menu.

**Comment by DGamer007** [comment]
Yes, I can confirm too that the issue I have explained above has been fixed. It's been 24 hours since my last fresh install of "VSCode Insiders" and the Context Menu options still works.

**Comment by DGamer007** [comment]
And It is gone again. Don't know how.
